setup.py

```diff
@@ -22,9 +22,6 @@
     'requests>=1.2.0'
 ]
 
-if sys.version_info < (3, 2):
-    requires.append('futures>=2.1.3')
```

Comment:
This will still need to be here to support 2.7. It was testing < 3.2, which didn't include 3.2.